packages/tailwind-config/theme/custom-scrollbar.css

```
@@ -118,10 +118,16 @@
 
 @utility modal-scrollbar {
   &::-webkit-scrollbar {
```

Stylelint (17.4.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)